From ddfe7252f67e6f35c9ea989f44ff3fb9997e9614 Mon Sep 17 00:00:00 2001
From: Wally Gleemer <Wally.Gleemer@windriver.com>
Date: Thu, 13 Mar 2008 20:34:15 -0700
Subject: [PATCH] cavium: RTC IRQ may not be eight.

Only force the RTC IRQ value to be eight (ISA default from
approx 1986) if we haven't already specified it elsewhere.

Signed-off-by: Wally Gleemer <Wally.Gleemer@windriver.com>
---
 include/asm-mips/mach-generic/mc146818rtc.h |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/include/asm-mips/mach-generic/mc146818rtc.h b/include/asm-mips/mach-generic/mc146818rtc.h
index 0b9a942..0c8359e 100644
--- a/include/asm-mips/mach-generic/mc146818rtc.h
+++ b/include/asm-mips/mach-generic/mc146818rtc.h
@@ -13,7 +13,9 @@
 #include <asm/io.h>
 
 #define RTC_PORT(x)	(0x70 + (x))
+#ifndef RTC_IRQ
 #define RTC_IRQ		8
+#endif
 
 static inline unsigned char CMOS_READ(unsigned long addr)
 {
-- 
1.5.5.1

